services/virtual-assistant/src/virtual-assistant/routes.py

```diff
@@ -27,38 +27,41 @@ async def health():
 @api_blueprint.route("/talk", methods=["POST"])
 @require_identity_header
 async def talk():
-    data = request.get_json()
+    request_body = await request.get_json()
```

Comment:
I'm confused here. 
Does this means that `watson` is actually talking to us?

Reply:
This is fetching the `request_body` the user is us.
I am actually updating this so that the validation gets done through `QuartSchema`, this step becomes much clearer when we do that.

Reply:
Updated to support quartschema now.